phys/module_microphysics_driver.F

```diff
@@ -457,6 +458,7 @@ SUBROUTINE microphysics_driver(                                          &
 
     REAL, DIMENSION( ims:ime , kms:kme , jms:jme ),INTENT(INOUT), OPTIONAL :: th_old,qv_old
     REAL,DIMENSION(ims:ime,kms:kme,jms:jme,num_scalar),INTENT(INOUT), OPTIONAL   :: scalar
+    REAL, DIMENSION(ims:ime,kms:kme,jms:jme,num_sbmradar),INTENT(INOUT) :: sbmradar
```

Comment:
I don't see how sbmradar comes into this module either. And this also needs to be declared 'optional'.

Reply:
@weiwangncar 

Indeed, the SBMRADAR needs to be defined as "optional".
It's dimension and definition are derived from the Registry file (at least to my knowledge):
Registry/registry.polrad